tests/test_pv_creation.py

```diff
@@ -21,7 +21,7 @@
 
 
 OCP = ocp.OCP(
-    kind='PersistentVolume', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+    kind='PersistentVolume', namespace=ENV_DATA['cluster_namespace']
```

Comment:
since the defaults is populated from yaml, I think leaving this as is makes it more readable?

Reply:
Ignore this one please, Its ENV here and not defaults.